README.md

```diff
@@ -1,36 +1,89 @@
 
-**⚠️ Experimental:** Early development stage, expect frequent changes and unresolved issues.
+> **Important**  
+> This software is in **alpha** status and ready for you to start experimenting with. However, we do not recommend using alpha software in production apps. Expect frequent changes as we add features and iterate based on feedback.
 
 ## Requirements
 
-- Install [Rustup](https://rustup.rs/)
+- Install [Rustup](https://rustup.rs/)
+- Install [Docker](https://www.docker.com/get-started/)
 
 ## Development
 
-Install other dependencies and start background services:
+Start Docker Desktop.
 
-```sh
-dev/up
-```
+- To install other dependencies and start background services:
 
-Run tests:
+  ```
+  dev/up
+  ```
 
-```sh
-dev/test
-```
+  Specifically, this command creates and runs an XMTP node in Docker Desktop.
+
+- To run tests:
+
+  ```
+  dev/test
+  ```
 
 ## Structure
 
-- `xmtp_cryptography` - cryptographic operations
-- `xmtp_proto` - Generated code for handling XMTP protocol buffers
-- `xmtp_networking` - API client for XMTP's GRPC API, using code from `xmtp_proto`
-- `xmtp` - the pure Rust implementation of the XMTP SDK, agnostic to any per-language or per-platform binding
-- `bindings_ffi` - FFI bindings for Android and iOS
-- `bindings_wasm` (unused) - wasm bindings
-- `bindings_js` (unused) - JS bindings
+- [`xmtp`](https://github.com/xmtp/libxmtp/tree/main/xmtp): Pure Rust implementation of XMTP APIs, agnostic to any per-language or per-platform binding
+- [`xmtp_cryptography`](https://github.com/xmtp/libxmtp/tree/main/xmtp_cryptography): Cryptographic operations
+- [`xmtp_networking`](https://github.com/xmtp/libxmtp/tree/main/xmtp_networking): API client for XMTP's gRPC API, using code from `xmtp_proto`
+- [`xmtp_proto`](https://github.com/xmtp/libxmtp/tree/main/xmtp_proto): Generated code for handling XMTP protocol buffers
+- [`examples/cli`](https://github.com/xmtp/libxmtp/tree/main/examples/cli): Example XMTP console client. Use the CLI to try out sending double ratchet messages on the XMTP `dev` network.
+- [`examples/android/xmtpv3_example`](https://github.com/xmtp/libxmtp/tree/main/examples/android/xmtpv3_example): Example Android app (in progress)
+- [`bindings_ffi`](https://github.com/xmtp/libxmtp/tree/main/bindings_ffi): FFI bindings for Android and iOS (in progress)
+- [`bindings_js`](https://github.com/xmtp/libxmtp/tree/main/bindings_js): JS bindings (in progress)
+- [`bindings_wasm`](https://github.com/xmtp/libxmtp/tree/main/bindings_wasm): Wasm bindings (in progress)
+
+## Key features of XMTP v3-alpha
+
+### Double ratchet messaging
+
+**What**: XMTP v3-alpha introduces the use of the [double ratchet algorithm](https://signal.org/docs/specifications/doubleratchet/) to keep past and future messages secure.
+
+**Why**: With double ratchet messaging, each message is encrypted by its own session key. If the session key a sender used to send a message becomes compromised:
```

Comment:
```suggestion
**Why**: With double ratchet messaging, each message is encrypted by its own key. If a message key becomes compromised:
```
As with the other comment, being a little bit specific about the session/message key terminology, sorry I didn't bring this up earlier.

Some degree of forward secrecy/PCS is actually maintained if any of the message/session/installation keys are compromised, but would require more nuance to explain below, so happy with this level of granularity as more detail can be found in the double ratchet spec.